fullstory/src/test/java/com/tealium/remotecommands/fullstory/FullStoryRemoteCommandTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.tealium.remotecommands.fullstory
+
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class FullStoryRemoteCommandTest {
+
+    @MockK
+    lateinit var mockFullStoryInstance: FullStoryCommand
+
+    lateinit var fullStoryRemoteCommand: FullStoryRemoteCommand
+
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this, relaxUnitFun = true)
+        fullStoryRemoteCommand = FullStoryRemoteCommand()
+
+        fullStoryRemoteCommand.fullStoryInstance = mockFullStoryInstance
```

Comment:
this now fails compilation because of the change to the `fullStoryInstance` property being converted to a `val`
probably easiest to either make it an `internal var` or just have it as a `private val` and inject it through the constructor